UPGRADE-2.0.md

```diff
@@ -112,6 +147,8 @@
       // ...
   }
   ```
+  
+  In general if you need to create your own instance of a `Client` we advise you to use the `ClientBuilder` instead since it does a lot for you internally.
```

Comment:
There is a space at the start of the line that should not exists. Also I would change the sentence to `The suggested way to create your own instance of the client is to use the provided builder that will take care of instantiating a few dependencies like the PSR-7 factories and the HTTP client`